### ISSUE TITLE
List appear in random order in drop down

### ISSUE THREAD

**Issue by Kirk-Hanson** [CGeorges/unrealsales-tracker#50]
I like many here I imagine am very grateful for this sites ability to catalogue items. I have around 50 lists with 1000 items broken down into categories and sub-categories, but for some reason the lists spawn in random order on the drop down, as oppose to alphabetically or even order-entered. The result is a jumbled mess that makes my head spin. Would it be a tough fix to add some rhyme or reason to the drop down order, or the ability to drag them around in the manager? Being able to rename them would be great too, although minor compared to confusing list order

Many thanks!
Kirk

**Comment by CGeorges** [comment]
Hi @Kirk-Hanson 
As a quick fix, all list are now ordered in Alphabetical order.

**Comment by Kirk-Hanson** [comment]
Thanks for getting back to me so quick

I logged out and back in and sure enough. Was that all I had to do to begin
with?

**Comment by CGeorges** [comment]
A refresh of the page should have been enough. Does it work for you?

**Comment by Kirk-Hanson** [comment]
It does, OCD is at peace :)

**Comment by Kirk-Hanson** [comment]
Hey Georges!

The wishlist on the main page is working well, thank you very much for
that. Today I started going through all the assets with the "move to list"
button, and the drop down list there is all jumbled up. I tried tuffing
through it, but with this number of items and the number of lists, it's
really hard. Also refreshing or relogging in wasn't working either. I took
a screenshot of what I mean.
